lms/templates/help_modal.html

```diff
@@ -40,81 +40,80 @@ <h2>
           <hr>
         </header>
 
-<%
-  discussion_tab = CourseTabList.get_discussion(course) if course else None
-  discussion_link = discussion_tab.link_func(course, reverse) if (discussion_tab and discussion_tab.is_enabled(course, user=user)) else None
-  has_user_registered = UserProfile.has_registered(user)
-%>
+    <%
+      discussion_tab = CourseTabList.get_discussion(course) if course else None
+      discussion_link = discussion_tab.link_func(course, reverse) if (discussion_tab and discussion_tab.is_enabled(course, user=user)) else None
+      has_user_registered = UserProfile.has_registered(user)
+    %>
 
-% if discussion_link and has_user_registered:
-    <p>${Text(_(
-      'For {strong_start}questions on course lectures, homework, tools, or materials for this course{strong_end}, post in the course discussion forum.'
-    )).format(
-      strong_start=HTML('<strong>'),
-      strong_end=HTML('</strong>')
-    )}</p>
-    <ul class="help-buttons">
-      <li><a class="help-modal-button"  href="${discussion_link}">
-        ${_('Post in the Forum')}
-      </a></li>
-    </ul>
-% endif
+    % if discussion_link and has_user_registered:
+        <p>${Text(_('For {strong_start}questions on course lectures, homework, tools, or materials for this course{strong_end}, post in the {link_start}course discussion forum{link_end}.')).format(
+          strong_start=HTML('<strong>'),
+          strong_end=HTML('</strong>'),
+          link_start=HTML('<a href="{url}" target="_blank">').format(
+            url=discussion_link
+          ),
+          link_end=HTML('</a>'),
+          )}
+        </p>
+    % endif
 
-    <p>${Text(_(
-      'Have {strong_start}general questions about {platform_name}{strong_end}? You can find lots of helpful information in the {platform_name} Help Center.'
-    )).format(
-      strong_start=HTML('<strong>'),
-      strong_end=HTML('</strong>'),
-      platform_name=static.get_platform_name()
-    )}</p>
+        <p>${Text(_('Have {strong_start}general questions about {platform_name}{strong_end}? You can find lots of helpful information in the {platform_name} Help Center.')).format(
```

Comment:
We should run it my Monica first, but I'd like to unfork these changes where we say "Help Center" instead of FAQ.
Not only is it a relatively minor change, but arguably FAQ is more apt.
Looking at the Zendesk, it's literally a list of frequently asked questions :)

This also means we could start using `marketing_link('FAQ')` for less forking of links across the platform.
This also lets us customize this across instances, which we'll probably need for the SCPD deployment.

![screen shot 2018-02-02 at 12 59 08](https://user-images.githubusercontent.com/7300975/35754670-1ba3f766-0819-11e8-96a6-5f92b6edd266.png)

Reply:
EdX's code is also superior in that it also links to the FAQ, instead of just mentioning it.